Lib/test/test_tools/test_i18n.py

```diff
@@ -516,6 +517,42 @@ def test_parse_keyword_spec(self):
                     parse_spec(spec)
                 self.assertEqual(str(cm.exception), message)
 
+    def test_normalize_multiline(self):
+        # required to set up normalize
+        class NormOptions:
+            width = 78
```

Comment:
```py
        options = SimpleNamespace(width=78)
```

and `from types import SimpleNamespace`. It's pretty useful to avoid inner classes like that just for readability.